services/player-service/adapter-persistence/src/main/java/com/kjeldsen/player/persistence/adapters/mongo/PlayerTrainingEventWriteRepositoryMongoAdapter.java

```diff
@@ -0,0 +1,21 @@
+package com.kjeldsen.player.persistence.adapters.mongo;
+
+import com.kjeldsen.player.domain.events.PlayerTrainingEvent;
+import com.kjeldsen.player.domain.repositories.PlayerTrainingEventWriteRepository;
+import com.kjeldsen.player.persistence.mongo.repositories.PlayerTrainingEventMongoRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@ConditionalOnProperty(name = "player.persistence.adapter", havingValue = "db")
```

Comment:
lets check if we can delete cache and stay by default with db. I do not think it is needed anymore